assets/app/view/game_page.rb

```diff
@@ -443,7 +448,11 @@ def render_game
       children << h(Game::EntityOrder, round: @round)
       unless @game.finished
         children << h(Game::Abilities, user: @user, game: @game)
-        children << h(Game::Pass, actions: current_entity_actions)
+        children << if @game.round.unordered? && hotseat_or_master
+                      h(Game::MasterPass)
```

Comment:
why is this needed?

Reply:
In rounds where multiple players may act at any point ("unordered"), while in hotseat or master_mode, I need to display multiple "pass" buttons - one per player that can still act.

Reply:
![RS_master](https://user-images.githubusercontent.com/8494213/162461314-dbd04a8d-ae03-4224-aba3-710f90d22460.png)

Reply:
Outside of master_mode:

![RS_nomaster](https://user-images.githubusercontent.com/8494213/162462858-f795c17b-3016-4a89-8581-d8d916695cda.png)